scripts/linters/pylint_extensions.py

```diff
@@ -1163,7 +1164,8 @@ def process_module(self, node):
             if line.count(string_indicator) >= 2:
                 continue
 
-            if re.search(br'[^/]/[^/]', line):
+            if re.search(br'[^/]/[^/]', line) and (
```

Comment:
Please move parenthesis to front of entire expression (like on line 1079)

Reply:
Done.